CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # gulp-static-i18n changelog
 
+## 0.0.2
+
+* Allow sourcing of PO files from Multiple directories
```

Comment:
changelogs are a good place to note breaking changes. Note that the config has changed from `localeDir` to `localeDirs`.  This change also needs to be reflected in the readme.